SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -254,14 +254,16 @@ const FString ZoningAttribute = DefaultServerWorkerType.ToString();
 // A list of components clients require on top of any generated data components in order to handle non-authoritative actors correctly.
 const TArray<Worker_ComponentId> REQUIRED_COMPONENTS_FOR_CLIENT_INTEREST = TArray<Worker_ComponentId>
 {
-	// Unclear why ACL is required. TODO(UNR-2768): Remove this requirement
-	ENTITY_ACL_COMPONENT_ID,
-
 	UNREAL_METADATA_COMPONENT_ID,
 	SPAWN_DATA_COMPONENT_ID,
 	RPCS_ON_ENTITY_CREATION_ID,
 	MULTICAST_RPCS_COMPONENT_ID,
-	NETMULTICAST_RPCS_COMPONENT_ID_LEGACY,
+	NETMULTICAST_RPCS_COMPONENT_ID_LEGACY
+};
+
+// A list of components clients require on entities they are authoritative over on top of the components already checked out by the interest query.
+const TArray<Worker_ComponentId> REQUIRED_COMPONENTS_FOR_CLIENT_AUTH = TArray<Worker_ComponentId>
```

Comment:
@m-samiec please let me know if there are any more gdk components required for client auth entities that aren't covered by interest (or @ the people who would know)

Reply:
this seems to work

Reply:
still requires current server qbi to be on for the servers to see the client rpc component but that will be fixed soon ™️